lib/gui/app.js

```diff
@@ -176,6 +176,11 @@ app.run(() => {
     ].join(' '));
 
     windowProgress.set(currentFlashState.percentage);
+
+    const DASH_UNICODE_CHAR = '\u2013';
+    const flashProgress = messages.info.flashProgress(currentFlashState);
+    const title = `${packageJSON.displayName} ${DASH_UNICODE_CHAR} ${flashProgress}`;
+    windowProgress.setTitle(title);
```

Comment:
But then how would we know whether to display "validating" or "flashing" in the title bar? Or perhaps `windowProgress.set()` should also have an extra parameter, specifying what state we're currently in? And thinking about it a bit more, maybe it makes sense for the title to also display "Finishing..." too, so that the user doesn't think that Etcher has actually finished before it already has, if they're _only_ watching the title-bar?